Support list of entities for generators (#57)
port-labs/Port-Ocean: changelog/improvement.2.md

> Displaying the extra object only for debug log level

Reviewer: ```suggestion
Changed the extra object messages log level from INFO to DEBUG
```
